frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -254,17 +254,28 @@ export const surveyLogic = kea<surveyLogicType>([
                 if (surveyId === 'new') {
                     return null
                 }
+                const createdAt = (survey as Survey).created_at
 
                 const surveysShownHogqlQuery = `select count(distinct person.id) as 'survey shown' from events where event == 'survey shown' and properties.$survey_id == '${surveyId}'`
                 const surveysDismissedHogqlQuery = `select count(distinct person.id) as 'survey dismissed' from events where event == 'survey dismissed' and properties.$survey_id == '${surveyId}'`
                 return {
                     surveysShown: {
                         kind: NodeKind.DataTableNode,
-                        source: { kind: NodeKind.HogQLQuery, query: surveysShownHogqlQuery },
+                        source: {
+                            kind: NodeKind.HogQLQuery,
+                            query: surveysShownHogqlQuery,
+                            filters: { dateRange: { date_from: dayjs(createdAt).format('YYYY-MM-DD') } },
```

Comment:
@liyiy I've added date ranges to all queries which should make them faster + more accurate